3-0-spring-framework/3-2-1-account-rest-api/src/main/java/com/bobocode/config/AccountRestApiInitializer.java

```diff
@@ -0,0 +1,20 @@
+package com.bobocode.config;
+
+import org.springframework.web.servlet.support.AbstractAnnotationConfigDispatcherServletInitializer;
+
+public class AccountRestApiInitializer extends AbstractAnnotationConfigDispatcherServletInitializer {
```

Comment:
@mathodcoast I had some ideas regarding refactoring, but I believe we should not do it right now. I have created a separate story since it's related to other exercises as well. (https://bobocode.atlassian.net/browse/GP-114)